android/app/src/main/java/network/mysterium/service/core/MysteriumAndroidCoreService.kt

```diff
@@ -50,12 +38,9 @@ class MysteriumAndroidCoreService : VpnService() {
         val openvpnBridge = Openvpn3AndroidTunnelSetupBridge(this)
         val wireguardBridge = WireguardAndroidTunnelSetup(this)
 
-        val logOptions = Mysterium.defaultLogOptions()
-        logOptions.filepath = filesPath
-        logOptions.logHTTP = false
         val options = Mysterium.defaultNetworkOptions()
 
-        mobileNode = Mysterium.newNode(filesPath, logOptions, options)
+        mobileNode = Mysterium.newNode(filesPath, options)
```

Comment:
Compilation error, method signature from 0.17.0 is:
`public static native MobileNode newNode(String var0, MobileLogOptions var1, MobileNetworkOptions var2) throws Exception;`